doc/devbox_setup.md

```diff
@@ -50,8 +52,9 @@ static const char* connectionString = "[device connection string]";
 ...and replace `[device connection string]` with a valid device connection string for a device registered with your IoT Hub. For more information, see the [samples section](#samplecode) below.
 
 Build the sample project.
+ 
 
-### Build the C SDK in Windows
+### Build the C SDK with CMake on Windows
 
 In some cases, you may want to build the SDK locally for development and testing purposes. First, take the following steps to generate project files:
```

Comment:
What about VS 2019 in the below commands?

Reply:
New generator for 2019, updated